src/components-examples/material/form-field/form-field-custom-control/form-field-custom-control-example.ts

```diff
@@ -115,6 +164,21 @@ export class MyTelInput implements ControlValueAccessor, MatFormFieldControl<MyT
     }
   }
 
+  autoFocusNext(
+    control: AbstractControl,
+    nextElement: HTMLInputElement | null
```

Comment:
why not `nextElement?: HTMLInputElement` instead

Reply:
resolved